deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -156,9 +156,9 @@ cephClusterSpec:
     # daysToRetain: 30
 
   # enable log collector, daemons will log on files and rotate
-  # logCollector:
-  #   enabled: true
-  #   periodicity: daily # one of: hourly, daily, weekly, monthly
+  logCollector:
+    enabled: true
+    periodicity: daily # one of: hourly, daily, weekly, monthly
 
```

```suggestion
maxLogSize: 500M # SUFFIX may be 'M' or 'G'. Must be at least 1M.
```
we need to add this for helm user too

---

> Ah sure i think we can have a similar change updated in `cluster-on-pvc.yaml`.
> 
> "also a thought, we were discussing adding something similar in krew plugin so can we add something in krew and let users decide if they require this advance debugging?"
> 
> > The current option will take care of the [1] aspect of #11151 by which coredumps will be generated by default. I was proposing to add [#11151 (comment)](https://github.com/rook/rook/issues/11151#issuecomment-1278925459) to krew plugin to automate the collection of coredump for a live process when needed.
> 
> I think we'll still need the `logCollector` to be enabled by default just in case a process terminates unexpectedly due to a bug/segfault which can be reported as an issue to the ceph team.
> 
AHh, got it thanks @gauravsitlani for explaining.

> "also, can we make periodicity hourly since this is upstream?"
> 
> > Sure sounds good.